docs/content/users/install/ddev-installation.md

```diff
@@ -137,9 +137,9 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     You can install DDEV on Windows three ways:
 
-    1. [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
-    2. [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
-    3. [Installing directly on traditional Windows](#traditional-windows) with an installer
+    1. **Most reliable, free, open-source:** [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
+    2. **Commercial, often problematic:** [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
+    3. **Old school:** [Installing directly on traditional Windows](#traditional-windows) with an installer
 
     **We strongly recommend using WSL2.** While its Linux experience may be new for some Windows users, it’s worth the performance benefit and common experience of working with Ubuntu and Bash.
```

Comment:
**We strongly recommend using WSL2.**

I now get that it means as compared to WSL1. How about

```suggestion
    **We strongly recommend updating your WSL to WSL2 for the best performance.**

Reply:
Thanks for reviewing! No, actually WSL1 is not usable with DDEV or Docker, and none of the instructions will work with WSL1, and no docker platform will work there. 

```suggestion
    **We strongly recommend using WSL2 for your Windows DDEV development environment.** While its Linux experience may be new for some Windows users, it’s worth the performance benefit and common experience of working with Ubuntu and Bash.
```